src/app/lib/analyticsUtils/index.test.js

```diff
@@ -439,28 +439,27 @@ describe('getAtUserId', () => {
   returnsNullWhenOffClient(getAtUserId);
 
   it('should return AT user id when found', () => {
-    Cookie.getJSON = jest.fn().mockReturnValue({ val: 'uuid' });
+    Cookie.get = jest.fn().mockReturnValue('uuid');
```

Comment:
oh is `val` part of the old `getJSON` api, if so my comment above is incorrect?

Reply:
I thought this but it's not actually the case. I have fixed the PR now